lte/gateway/c/oai/common/conversions.h

```diff
@@ -187,6 +200,12 @@
     BUFFER_TO_INT16((aSN)->buf, x);                                            \
   } while (0)
 
+#define OCTET_STRING_TO_INT24(aSN, x)                                          \
+  do {                                                                         \
+    DevCheck((aSN)->size == 2, (aSN)->size, 0, 0);                             \
```

Comment:
should not this be `== 3` rather than `== 2`.

Reply:
yes, correct Thanks